.github/workflows/build-lint-test.yml

```diff
@@ -8,11 +8,17 @@ jobs:
     name: Prepare
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
       - name: Use Node.js
         uses: actions/setup-node@v4
         with:
-          node-version-file: '.nvmrc'
+          node-version: 'lts/*'
```

Comment:
Should we not prefer the version in `.nvmrc`?

Reply:
According to the PR description, we are now running this first before checking out, so we don't have access to `.nvmrc` anymore.